Etch.OrchardCore.News.csproj

```diff
@@ -23,6 +23,7 @@
   <ItemGroup>
     <PackageReference Include="OrchardCore.ContentManagement" Version="1.5.0" />
     <PackageReference Include="OrchardCore.DisplayManagement" Version="1.5.0" />
+    <PackageReference Include="OrchardCore.Lucene" Version="1.4.0" />	  
```

Comment:
Change to target 1.5.0. We'll change it separately when we do a tag that targets 1.4.0,

Reply:
there are only pre-release versions for 1.5.0, is that ok? i just went with the stable 1.4